### PR TITLE
Support 2020-06-01 API version and add ppg parameter

### PR DESCRIPTION
**Pull request by tilnl**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/aks-preview/azext_aks_preview/_help.py [hunk 1]
```diff
@@ -172,6 +172,9 @@
         - name: --vnet-subnet-id
           type: string
           short-summary: The ID of a subnet in an existing VNet into which to deploy the cluster.
+        - name: --ppg
```
Comment (zqingqing1) [inline src/aks-preview/azext_aks_preview/_help.py:175]: --ppg-id?

File: src/aks-preview/azext_aks_preview/_params.py [hunk 2]
```diff
@@ -83,6 +83,7 @@ def load_arguments(self, _):
         c.argument('pod_cidr')
         c.argument('service_cidr')
         c.argument('vnet_subnet_id', type=str, validator=validate_vnet_subnet_id)
+        c.argument('ppg')
```
Comment (zqingqing1) [inline src/aks-preview/azext_aks_preview/_params.py:86]: Do you need to add some validation fo the ppg string?

Comment (tilnl) [reply src/aks-preview/azext_aks_preview/_params.py:86]: There is validation in rp sync code.

File: src/aks-preview/azext_aks_preview/_help.py (hunk 1 above)
Comment (tilnl) [reply src/aks-preview/azext_aks_preview/_help.py:175]: According to cli command guidelines https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#argument-naming-conventions 
ID Arguments:
Arguments that expect ARM IDs should omit -id but call out that an ARM ID is allowed in the help text. These are often used as part of the "name or ID" convention (i.e: --storage-account can often accept a storage account name OR ARM ID).

**Comment by JunSun17** [comment]
LGTM.

File: src/aks-preview/azext_aks_preview/_help.py [hunk 3]
```diff
@@ -484,6 +487,9 @@
...
+          type: string
+          short-summary: The ID of a PPG.
```
Comment (gtxistxgao) [inline src/aks-preview/azext_aks_preview/_help.py:492]: use full name of PPG may be better.

Comment (tilnl) [reply src/aks-preview/azext_aks_preview/_help.py:492]: Following vmss, use -ppg.

**Pull request merged**